dspace-server-webapp/src/main/java/org/dspace/app/rest/authorization/impl/CanCreateSubmission.java

```diff
@@ -0,0 +1,114 @@
+package org.dspace.app.rest.authorization.impl;
+
+import java.sql.SQLException;
+import java.util.Iterator;
+
+import org.dspace.app.rest.authorization.AuthorizationFeature;
+import org.dspace.app.rest.authorization.AuthorizationFeatureDocumentation;
+import org.dspace.app.rest.model.BaseObjectRest;
+import org.dspace.app.rest.model.CollectionRest;
+import org.dspace.app.rest.model.SiteRest;
+import org.dspace.authorize.service.AuthorizeService;
+import org.dspace.content.Item;
+import org.dspace.content.service.ItemService;
+import org.dspace.core.Context;
+import org.dspace.eperson.EPerson;
+import org.dspace.eperson.Group;
+import org.dspace.services.ConfigurationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import com.google.common.collect.Streams;
+
+
+/**
+ * This authorization feature is a bit different from 'canSubmit'.
+ * In this feature we rather check if the user can make many submissions, only one or none.
+ * This is useful in cases where a student can only make one submission (for a master thesis for ex.)
+ * but where an administrator account can submit multiple times.
+ */
+@Component
+@AuthorizationFeatureDocumentation(name = CanCreateSubmission.NAME, 
+    description = "It can be used to verify if a user has the rights to create a new submission in the repository.")
```

Comment:
.. if user is allowed to create a new submission.... ?? (my english isn't perfect ;-))